assets/src/i18n/en-GB.json

```diff
@@ -81,6 +81,9 @@
     "title": "Braintree by PayPal",
     "emptyStateTitle": "No braintree transaction found",
     "emptyStateDescription": "This order was not handled through Braintree",
+    "invalidEmptyStateTitle": "No braintree transaction found",
+    "invalidEmptyStateDescription": "Transaction was placed by another account or sandbox mode enabled | Transaction was placed by another account or sandbox mode disabled",
```

Comment:
Why is the text here twice?

Reply:
true, was thinking of a different title but reused the existing one - let me squash that